sale_comment_propagation/models/stock.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Ainara Galdona - AvanzOSC
+# Copyright 2015 Oihane Crucelaegui - AvanzOSC
+# Copyright 2015 Esther Martín <esthermartin@avanzosc.es> - Avanzosc S.L.
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, models, fields
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    sale_comment = fields.Text(string='Internal comments')
+    sale_propagated_comment = fields.Text(
+        string='Propagated internal comments')
+
+    @api.one
+    @api.onchange('partner_id')
```

Comment:
> Don't use @api.one with @api.onchange or it will crash.
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0

(besides @api.one is deprecated)